docs/user_manual.md

```diff
@@ -204,8 +126,8 @@ spec:
     name: ${your-service-name}
     labels:
       version: canary       # These map is used to label these canary instances
-  deploy:                   # K8s native deployment spec contents
-    replicas: 2
+  deploy:                   # K6s native deployment spec contents
+    replicas: 0
```

Comment:
debug change?

Reply:
Thanks for your careful review. Updated!